src/components/map/tools/ExtendRoutePathTool.ts

```diff
@@ -39,7 +39,8 @@ class ExtendRoutePathTool implements BaseTool {
             RoutePathStore!.setNeighborToAddType(queryResult!.neighborToAddType);
 
         } catch (ex) {
-            ErrorStore.addError((ex as Error).message);
+            ErrorStore.addError(
+                'Unable to process network node click using ExtendRoutePathTool', ex);
```

Comment:
Use finnish

Reply:
oh

Reply:
true dat

Reply:
Do you have a available translation for: Unable to process network node click using ExtendRoutePathTool

Reply:
Perhaps `Solmusta (soltunnus: ${properties.soltunnus} alkavien linkkien haku epäonnistui`

But hmm, seems like routePathLinkServie already does a try catch... maybe we should remove it if we decided to try catch outside of service